helmfile.d/lists/images.yaml

```
@@ -0,0 +1,6 @@
+---
+images:
+  ingressNginxChart:
+    controller: registry.k8s.io/ingress-nginx/controller-chroot:v1.12.1@sha256:90155c86548e0bb95b3abf1971cd687d8f5d43f340cfca0ad3484e2b8351096e
+    defaultBackend: registry.k8s.io/defaultbackend-amd64:1.5
+    admissionWebhooks: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.5.2@sha256:e8825994b7a2c7497375a9b945f386506ca6a3eda80b89b74ef2db743f66a5ea
```

Might be a better description, as it generates and patches in the certificate for the webhook which is in the controller.
```suggestion
    admissionWebhookPatch: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.5.2@sha256:e8825994b7a2c7497375a9b945f386506ca6a3eda80b89b74ef2db743f66a5ea
```

---

The same image is used in both the `job-createSecret` and `job-patchWebhook` files, and they refer to it in the chart as `.Values.controller.admissionWebhooks.patch.image`.

---

Done 421b0bcf